17_1_shape.js

```diff
@@ -0,0 +1,64 @@
+const ctx = document.querySelector('canvas').getContext('2d');
```

Comment:
Maybe, instead of `ctx` name it just `canvas`? :)

Reply:
Oh, you are using `getContext` also. So maybe `context`? Using shortcuts isn't a good pattern usually.

Reply:
it's a pattern that canvas users will recognize, though